### PR TITLE
[#IOPSC-121] Refactor documents ingestion

### PR DESCRIPTION
**Pull request by LoFrance**
This is the new handler for new data fields.

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: OnServiceChange/handler.ts [hunk 1]
```diff
@@ -161,14 +172,22 @@ export const createUpsertSql = (dbConfig: IDecodableConfigPostgreSQL) => (
...
+      "scope",
+      "serviceMetadata",
+      "subscriptionAccountEmail",
       "subscriptionAccountId",
       "subscriptionAccountName",
       "subscriptionAccountSurname",
-      "subscriptionAccountEmail"
+      "version"
     ])
     .whereRaw(`"${dbConfig.DB_TABLE}"."version" < excluded."version"`)
```
Comment (balanza) [inline OnServiceChange/handler.ts:192]: Use `<=` so we can update records if we replay change feed
```suggestion
    .whereRaw(`"${dbConfig.DB_TABLE}"."version" <= excluded."version"`)
```

File: OnServiceChange/handler.ts [hunk 2]
```diff
@@ -198,9 +226,12 @@ export const storeDocumentApimToDatabase = (
           telemetryClient
         ),
         TE.chainW(apimUser =>
+          TE.of({ apimUser, quality: getQuality(retrievedDocument) })
+        ),
```
Comment (balanza) [inline OnServiceChange/handler.ts:230]:
```suggestion
        TE.map(apimUser =>
          ({ apimUser, quality: getQuality(retrievedDocument) })
        ),
```

Comment (LoFrance) [reply OnServiceChange/handler.ts]: Done with d275768

File: OnServiceChange/handler.ts (hunk 1 above)
Comment (LoFrance) [reply OnServiceChange/handler.ts]: Fix with 5530311

**Pull request merged**